Library/Homebrew/exceptions.rb

```diff
@@ -65,6 +65,25 @@ def to_s; <<-EOS.undent
   end
 end
 
+class TapFormulaAmbiguityError < RuntimeError
+  attr_reader :name, :paths, :formulae
+
+  def initialize name, paths
+    @name = name
+    @paths = paths
+    @formulae = paths.map do |path|
+      path.to_s =~ HOMEBREW_TAP_PATH_REGEX
+      "#{$1}/#{$2.sub("homebrew-", "")}/#{path.basename(".rb")}"
+    end
+
+    super <<-EOS.undent
+      Multiple formulae found: #{formulae.map { |f| "\n       * #{f}" }.join}
```

Comment:
Maybe `Formulae found in multiple taps:` or `multiple locations` or something? I dunno.